pool/hub.go

```diff
@@ -244,22 +245,22 @@ func NewHub(cancel context.CancelFunc, hcfg *HubConfig) (*Hub, error) {
 	h.poolDiffs = NewDifficultySet(h.cfg.ActiveNet, powLimit, maxGenTime)
 
 	pCfg := &PaymentMgrConfig{
-		db:                     h.cfg.DB,
-		ActiveNet:              h.cfg.ActiveNet,
-		PoolFee:                h.cfg.PoolFee,
-		LastNPeriod:            h.cfg.LastNPeriod,
-		SoloPool:               h.cfg.SoloPool,
-		PaymentMethod:          h.cfg.PaymentMethod,
-		PoolFeeAddrs:           h.cfg.PoolFeeAddrs,
-		WalletAccount:          h.cfg.WalletAccount,
-		WalletPass:             h.cfg.WalletPass,
-		GetBlockConfirmations:  h.getBlockConfirmations,
-		GetTxConfNotifications: h.getTxConfNotifications,
-		FetchTxCreator:         func() TxCreator { return h.nodeConn },
-		FetchTxBroadcaster:     func() TxBroadcaster { return h.walletConn },
-		CoinbaseConfTimeout:    h.cfg.CoinbaseConfTimeout,
-		SignalCache:            h.SignalCache,
-		HubWg:                  h.wg,
+		db:                    h.cfg.DB,
+		ActiveNet:             h.cfg.ActiveNet,
+		PoolFee:               h.cfg.PoolFee,
+		LastNPeriod:           h.cfg.LastNPeriod,
+		SoloPool:              h.cfg.SoloPool,
+		PaymentMethod:         h.cfg.PaymentMethod,
+		PoolFeeAddrs:          h.cfg.PoolFeeAddrs,
+		WalletAccount:         h.cfg.WalletAccount,
+		WalletPass:            h.cfg.WalletPass,
+		GetBlockConfirmations: h.getBlockConfirmations,
+		FetchTxCreator:        func() TxCreator { return h.nodeConn },
+		FetchTxBroadcaster:    func() TxBroadcaster { return h.walletConn },
+		CoinbaseConfTimeout:   h.cfg.CoinbaseConfTimeout,
+		Cancel:                h.cancel,
```

Comment:
Cancel is not used anywhere. Should either be used or removed.

Reply:
Noted, removing.